src/components/Popup/ExportPage.vue

```diff
@@ -43,7 +46,8 @@ export default Vue.extend({
       unsupportedAccounts: hasUnsupportedAccounts(exportData),
       exportFile: getBackupFile(exportData),
       exportEncryptedFile: getBackupFile(exportEncData, key),
-      exportOneLineOtpAuthFile: getOneLineOtpBackupFile(exportData)
+      exportOneLineOtpAuthFile: getOneLineOtpBackupFile(exportData),
+      currentlyEncrypted: this.$store.getters["accounts/currentlyEncrypted"]
```

Comment:
Getters should be computed

Reply:
When user click "Encrypted" and input new password, that value could be changed

Reply:
Putting the getter in `computed` will still cause it to update properly, it just looks at the reactive dependencies which will work because of vuex. We are already doing this in other places like filter logic:

https://github.com/Authenticator-Extension/Authenticator/blob/ce0e82962353b6e73dd349294172ace512f00131/src/components/Popup/MainBody.vue#L57-L62

Reply:
Updated